source/Willow-Auth0/Auth0UniversalLoginAuthenticationFilter.class.st

```diff
@@ -13,22 +13,50 @@ Class {
 		'knownStates',
 		'jsonWebSignatureProvider',
 		'knownNonces',
-		'scheme'
+		'scheme',
+		'optionalOrganizationId',
+		'optionalAudience'
 	],
 	#category : #'Willow-Auth0'
 }
 
 { #category : #'instance creation' }
-Auth0UniversalLoginAuthenticationFilter class >> at: aDomain for: aClientId redirectingTo: aRedirectionURI nonceProvidedBy: aBlock JWSProvidedBy: aDyadicBlock [
+Auth0UniversalLoginAuthenticationFilter class >> at: aDomain for: aClientId in: aOrganizationId addressedTo: anAudience redirectingTo: aRedirectionURI nonceProvidedBy: aBlock JWSProvidedBy: aDyadicBlock [
+
+	^ self at: aDomain
+		for: aClientId
+		optionalIn: (Optional containing: aOrganizationId)
+		optionalAddressedTo: (Optional containing: anAudience)
+		redirectingTo: aRedirectionURI
+		nonceProvidedBy: aBlock
+		JWSProvidedBy: aDyadicBlock
+]
+
+{ #category : #private }
+Auth0UniversalLoginAuthenticationFilter class >> at: aDomain for: aClientId optionalIn: anOptionalOrganizationId optionalAddressedTo: anOptionalAudience redirectingTo: aRedirectionURI nonceProvidedBy: aBlock JWSProvidedBy: aDyadicBlock [
```

Comment:
These are a lot of external collaborators. Can't we find a way to compose them somehow? Can we add an object to represent part of them, then delegate the behavior inside the class to it?